src/js/drop-down.js

```diff
@@ -59,8 +59,17 @@ class DropDown {
 				return;
 			}
 
-			// Bail if there's no parent menu to toggle.
-			const parentMenu = event.target.closest('[data-o-header-services-level="1"]');
+			// Bail if there's no parent menu to toggle. Note: IE11 does not
+			// support `Array.prototype.find()` or `Element.closest()`,
+			// adding a polyfill requirement is a breaking change
+			let parentMenu;
+			for (const navItem of this.navItems) {
+				if (navItem.contains(event.target)) {
+					parentMenu = navItem;
+					break; // found the parent menu
+				}
+			}
```

Comment:
```suggestion
			let parentMenu;
			for (let itemIndex = 0; itemIndex < this.navItems.length; itemIndex++) {
				const navItem = this.navItems[itemIndex];

				if (navItem.contains(event.target)) {
					parentMenu = navItem;
					break; // found the parent menu
				}
			}
```

Reply:
Thanks I was actually expecting that `for...of` to be transpilled away, I was just looking into whether that was a reasonable expectation. I think [Babel does](https://babeljs.io/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&spec=false&loose=false&code_lz=GYewTgBAFAxiB2BnALhAlhEwIG0DkAhngDQR4BGeAugJQQDeAUBBHEiADYCmAdByAHMoaGgG5GAXyA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=script&lineWrap=false&presets=env%2Ces2015&prettier=false&targets=&version=7.11.6&externalPlugins=)

Reply:
yeah, if our policy is shipping javascript and expected the consumer to transpile for their platform then i think it is a reasonable expectation.

it would be nice if we could expect users to provide the correct polyfills too, by running something like js-features-analyser on their bundle.

Reply:
>yeah, if our policy is shipping javascript and expected the consumer to transpile for their platform then i think it is a reasonable expectation. (and i believe that is our policy)

Just has a look, it is! [JavaScript in components may use ES6 syntax, which product developers are expected to transpile to be ES5 if required](https://origami.ft.com/spec/v1/components/javascript/#feature-stability-and-polyfills). Although we dont specify a tool. Looks like Babel would handle it fine but via [swc](https://github.com/swc-project/swc) an error is thrown so I can't test this in IE as is -- there's [an swc issue here but without discussion](https://github.com/swc-project/swc/issues/1019). Think I'll just commit the`for` loop but wonder if any other components have been caught out by our shift from Babel.

>it would be nice if we could expect users to provide the correct polyfills too, by running something like js-features-analyser on their bundle.

Yeah it would be nice if it was automated in projects for minor/patch releases